### PR TITLE
Support Style-Based Global Appearance Flow for Virtual Try-On

### PR DESCRIPTION
**Pull request by FerryHuang**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/en/device/npu.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+```shell
+python tools/train.py configs/edsr/edsr_x2c64b16_1xb16-300k_div2k.py
+```
+
+## Models Results
+
+|                                           Model                                            | Dataset | PSNR  | SSIM | Download                                                                                       |
+| :----------------------------------------------------------------------------------------: | ------- | :---: | :--- | :--------------------------------------------------------------------------------------------- |
+| [edsr_x2c64b16_1x16_300k_div2k](https://github.com/open-mmlab/mmediting/blob/main/configs/edsr/edsr_x2c64b16_1xb16-300k_div2k.py) | DIV2K   | 35.83 | 0.94 | [log](https://download.openmmlab.com/mmediting/device/npu/edsr/edsr_x2c64b16_1xb16-300k_div2k.log) |
```
Comment (LeoXing1996) [inline docs/en/device/npu.md:23]: Cool, you add a doc for NPU device. But this log file does not exit on OpenMMLab's file server. You can email this file to me, and then I can upload the log.

**Comment by LeoXing1996** [comment]
@FerryHuang. Thanks for your contribution! This project will add a new kind of task to MMEditing!
To merge this PR, please:
- [ ] Sign CLA
- [ ] Update the branch with the latest `dev-1.x` and fix the lint error. (Unit test error will be fixed automatically after branch updating)

BTW, to make this PR more attractive, you can add some try-on results to the README file!

**Comment by FerryHuang** [comment]
I have merged the latest dev branch and create another pr :)

**Comment by LeoXing1996** [comment]
Close since duplicate with #1786.

**Pull request closed without merging**